src/styles/_catalog.scss

```
@@ -20,3 +20,10 @@
     margin-right: 48px;
   }
 }
+$screen-sizes: "2" 488px, "3" 768px, "4" 1024px;
```

list each option on a new line for better readability
```suggestion
$screen-sizes:
  "2" 488px,
  "3" 768px,
  "4" 1024px;
```